src/init/mountEndpoints.ts

```diff
@@ -0,0 +1,14 @@
+import { Router } from 'express';
+
+import { Payload } from '../index';
+import { Endpoint } from '../config/types';
+
+function mountEndpoints(router: Router, endpoints: Endpoint[], ctx: Payload): void {
+  endpoints.forEach((endpoint) => {
+    endpoint.handlers.forEach((handler) => {
+      router[endpoint.method].call(router, endpoint.route, handler.bind(ctx));
```

Comment:
I think you are binding `this` to expose Payload within these handlers. But, Payload is _**already available**_ and bound to `req` at this stage via Payload middleware further up the stack (can access via `req.payload`). So, binding `this` is not necessary IMO.

Reply:
> Everything looks great! I believe @DanRibbens is working on documentation here and I have a few changes—namely just removing `this` binding as it is not necessary.

I've already made this change and am finishing up some other improvements. @GeorgeyB standby, I'm going to push soon.

Reply:
@jmikrut This was to maintain consistency with the Payload's existing handlers where `this` is bound to the Payload context, but you're absolutely right, it's unnecessary if it's already accessible in `req`.

@DanRibbens Nice.

Reply:
I ran out of time working through these changes. I'll try and pick it up over the weekend to wrap this one up.